components/tidb_query/src/aggr_fn/impl_bit_op.rs

```diff
@@ -59,20 +56,7 @@ impl<T: BitOp> AggrFnDefinitionParserBitOp<T> {
 impl<T: BitOp> super::AggrDefinitionParser for AggrFnDefinitionParserBitOp<T> {
     fn check_supported(&self, aggr_def: &Expr) -> Result<()> {
         assert_eq!(aggr_def.get_tp(), T::tp());
-
         super::util::check_aggr_exp_supported_one_child(aggr_def)?;
```

Comment:
Notice that currently copr-test does not contain aggregation SQLs, so this change might not be covered?

Reply:
This BUG is detected by tidb-test randgen cases.